.github/workflows/pr.yml

```diff
@@ -287,7 +287,7 @@ jobs:
       JOB_NAME: functional_podman_ubuntu
       GOPOGH_RESULT: ""
       SHELL: "/bin/bash" # To prevent https://github.com/kubernetes/minikube/issues/6643
-    runs-on: ubuntu-20.04
+    runs-on: [ubuntu-20.04, !arm64]
```

Comment:
we could do is 
`!self-hosted` so it never runs on self-hosted ones

Reply:
It's still draft, trying to test different options :)